package.json

```diff
@@ -17,10 +22,23 @@
   },
   "devDependencies": {
     "autoprefixer": "^10.4.12",
-    "eslint": "8.24.0",
+    "eslint": "^8.0.1",
     "eslint-config-next": "12.3.1",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-n": "^15.0.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-react": "^7.31.8",
```

Comment:
Could you check if there is a plugin for `nextjs`. Check [this](https://nextjs.org/docs/basic-features/eslint) out.
I believe we should be using `eslint-plugin-next`

Reply:
Thanks. It's working fine now.